src/Templater.js

```diff
@@ -20,12 +20,53 @@ export default class Templater {
 
         this._built = false;
 
-        this._i18n = config.i18n;
+        this._i18n = config.i18n || this._default_i18n();
```

Comment:
I think we should change this to transformer or mapper or something so it doesn't need to be handling just i18n. But perhaps any sort of key to value conversions.